logic/calendar.py

```diff
@@ -0,0 +1,35 @@
+import urllib.request, json
+from os import getenv
+
+class CalendarImpl:
+    def command_parse(text: str):
+        command = text.split(" ")        
+
+        title = command[0]
+        dateindex = int(command.index("-d") + 1)
+        date = command[dateindex]
+        locationindex = int(command.index("-l") + 1)
+        location = command[locationindex]
```

Comment:
```suggestion
    def _create_post_data(title: str, date: str, location: str) -> bytes:
```
- -d -l って必要ないと思う。
コマンドの引数部分は help に乗せれば良い。
- 関数名を command_parse ってするなら、parse した結果を返すべき
今の動作に正しく関数に名前を付けるならこうかな
- プライベートメソッドにするつもりなら、先頭にアンダーバーをつけよう